specification/cognitiveservices/OpenAI.Inference/main.tsp

```diff
@@ -38,9 +38,9 @@ namespace Azure.OpenAI;
 #suppress "@azure-tools/typespec-azure-core/documentation-required" "https://github.com/Azure/typespec-azure/issues/3107"
 enum ServiceApiVersions {
   @useDependency(Azure.Core.Versions.v1_0_Preview_1)
-  v2022_12_01: "2022-12-01",
+  v2023_05_15: "2023-05-15",
   @useDependency(Azure.Core.Versions.v1_0_Preview_1)
-  v2023_03_15_Preview: "2023-03-15-preview",
+  v2023_06_01_Preview: "2023-06-01-preview",
```

Comment:
shouldn't we be adding values to the versions enum instead of modifying? Also, are the other endpoints absolutely the same for `v2022_12_01` and `v2023_03_15_Preview` ?

Reply:
oops, should have been 2023-05-15 (no preview)

Reply:
the changes are just additive, we may want to keep all the non-preview versions, but preview versions sometime disappear.